src/components/GroupChatDetails/GroupChatDetails.tsx

```diff
@@ -1,7 +1,7 @@
 import { Paper, TableBody } from '@mui/material';
 import React, { useRef, useState } from 'react';
 import { Button, Form, ListGroup, Modal } from 'react-bootstrap';
-import styles from './GroupChatDetails.module.css';
+import styles from '../../style/app.module.css';
```

Comment:
Remove the codecov ignore statements in this file and add tests to get the code coverage as close to 100% as possible.